### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jnxdd**
Added additional examples and use cases with a few links to bring more of what we do to life.

## Description

(Add description of main change here - feel free to modify overall template as needed, deleting/adding sections.)

### Linked GitHub Tickets

- Closes #XXX
- Closes #YYY

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,45 @@
 
 [![Verify Main Branch](https://github.com/getmash/mash-js/actions/workflows/main.yml/badge.svg?branch=main)](https://github.com/getmash/mash-js/actions/workflows/main.yml)
 
-Our objective at Mash is to make it easy for you to earn for your amazing creations. From enabling your users to pay-you-as-they-enjoy, boost-tip with a click or tap, and as they engage with you and the community. All for any amount, in your control.
+# Overview
 
-## How do I get started?
+Mash is the monetization platform for developers, indie-hackers, creators, builders who want to earn for their content, apps & experiences based on the value that they provide and learn what users love most. You can charge any amount, add usage-based-pricing to monetize any event, action or click without requiring your users to make a large-up-front purchases or input credit card details every time. Mash unlocks the fastest way to monetize experiences and get donations from your users, because their money is immediately available in your experience, and they can set a budget to auto-pay or auto-donate without interruption. It's your pricing, your experiences, your decisions, your control.
```
Comment (cartonioh) [inline README.md:15]: typo: `a large-up-front purchases`
```suggestion
a large up-front purchase
```

**Review by mdedys — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cartonioh) [inline README.md:15]: typo and nit: a large-up-front purchases

suggestion:
a large up-front purchase

File: README.md [hunk 2]
```diff
@@ -10,12 +10,45 @@
...
+
+* **Articles & Blog Posts** – From a review of your favourite products, to a how-to guide or recipe. Hide some content in a section or an entire post.
+* **Hidden Page** – ask users to contribute to see everything on a secret page of anything from behind the scenes footage, or pre-released content.
+* **Downloads** – Share high-quality assets for a fee. For cheat sheets, mods, image assets, filter packs and more.
+* **Video & Audio** – For premium deep-dives, behind the scenes content and more
+* **Links** – Have a great resource to share but want users to contribute a small bit ask users to contribute for premium links to resources.
+* **Books & Guides** – Authors are monetizing their experiences per chapter or page. You can get started easily with our open-source [Book Starter Gatsby Repo](https://github.com/getmash/mash-lightning-book-starter).
+
+## Boosts & Donations
+* **Boosts** – Let your users one-click-donate to you, without requiring a confirmation. You'll learn what they love, and they'll get the satisfcation of supporting you.
```
Comment (cartonioh) [inline README.md:50]: typo: satisfcation

suggestion: satisfaction

Comment (mdedys) [inline README.md:50]:
```suggestion
* **Boosts** – Let your users one-click-donate to you, without requiring a confirmation. You'll learn what they love, and they'll get the satisfaction of supporting you. 
```

**Pull request merged**